CleanProFinder.Web/src/app/modules/shared/components/card-design/card-design.component.scss

```diff
@@ -0,0 +1,30 @@
+.card-design {
+  border-radius: 40px;
+  height: 100%;
+  aspect-ratio: 630/984;
+  padding: 5%;
+  background-image: url('../../../../../assets/images/card-design-bg.png');
```

Comment:
Make this components reusabel. i mean we wont dublcate it for every screen. Take imahe url, title and text as input in component